tenprintcover.py

```diff
@@ -32,6 +32,9 @@
 
 import cairocffi as cairo
 
+PY2 = sys.version_info[0] == 2
+if PY2:
+    FileNotFoundError = IOError
```

Comment:
running pylint found this error. I don't think your proposed solutions works, but 

``` except (IOError, FileNotFoundError):``

might, depending on the order of evaluation, but that seems hackier.